### DISCUSSION TITLE
关于切换配置文件快速生效的疑问请教大佬。。。

### DISCUSSION THREAD

**Discussion by drw2005** [nikkinikki-org/OpenWrt-nikki#200] [General]
安装在openwrt软路由上，最新版本1.8.3 alpha-e33d4a4：
1、看wiki里说明，勾选了快速重载后，切换配置文件就直接生效？但是试了下，好像并不是，而是需要点击保存并应用后才能生效。
2、切换配置文件后，核心状态也没有运行，必须点击一下重载服务，状态才显示运行中
3、又发现个问题，显示配置文件切换成功了，而且查看用于“启动的配置文件”里也显示的正确，但是打开面板，居然还是上一个配置文件的节点。

请教大佬，这个配置文件不能切换后即时生效吗？

**Comment by morytyann** [comment]
1. 肯定需要保存并应用啊，Wiki写的是`修改插件配置会触发插件的重载`，你修改插件配置，肯定是保存并应用才能生效
2. 每五秒刷新一次，你可以等一会再看看
3. 这个没遇到过，可能是你没刷新面版，也可能是有什么问题，关掉快速重载再试试

**Comment by drw2005** [comment]
1、那可能我对快速重载没理解清楚，这么说来，这个勾不勾选影响不大？
2、似乎发现个BUG，刚才把两个机场订阅都清除掉了，配置文件那里显示为空，然后重新点击添加没有反应了，找了个yaml文件上传上去保存应用，再点击添加才正常显示。
3、不知道为啥，我这边一直显示第一次订阅的配置文件，刚才都清除掉，再重新订阅另一个机场，启动文件也显示获取有节点，但是面板显示的还是上一个配置，神奇。清缓存，换电脑，换浏览器都是如此。这个订阅文件保存在哪里？我直接删除试试。

**Reply by drw2005** [reply]
4、重启路由后，状态显示是运行中，但是似乎并没有生效，来回切换了一下节点，正常了。

**Reply by morytyann** [reply]
1. 确实影响不大
2. 我看看能修不
3. 这个不知道是什么问题，我测试正常，之前也没遇到过，如果你有多个配置文件且需要切换，建议关闭快速重载
